### PR TITLE
net-irc/weechat: add 4.0.5; add ruby 3.2 compatibility; add myself as a maintainer

### PR DESCRIPTION
**Pull request by sevz17**
~~Thinking about becoming a maintainer...~~

### CODE REVIEW

File: net-irc/weechat/metadata.xml [hunk 1]
```diff
@@ -9,6 +9,10 @@
 		<email>zoltan@sinustrom.info</email>
 		<name>Zoltan Puskas</name>
 	</maintainer>
+	<maintainer type="person" proxied="yes">
```
Comment (Flowdalic) [inline net-irc/weechat/metadata.xml:12]: Always also add the proxy-maint project when adding a proxied maintainer.

Comment (Flowdalic) [inline net-irc/weechat/metadata.xml:12]: Hmm, I thought we had a rule that the proxy-maint project should also be added if a proxied maintainer was added.

Comment (sevz17) [reply net-irc/weechat/metadata.xml:12]: Me too, but... https://github.com/gentoo/gentoo/pull/33064#discussion_r1337426042

Comment (Flowdalic) [reply net-irc/weechat/metadata.xml:12]: Yes, there it is: https://wiki.gentoo.org/wiki/Project:Proxy_Maintainers/User_Guide#Proxied_maintainer_in_metadata.xml

Could you add the proxy-maint project too please?

Comment (sevz17) [reply net-irc/weechat/metadata.xml:12]: Done

**Pull request closed without merging**